docs/3.0.0-alpha.x/guides/deployment.md

```diff
@@ -335,7 +335,7 @@ These instructions assume that you have already created a **Strapi** project, an
 
 **On your local computer:**
 
-You will need to update the `database.json` file to configure Strapi to connect to the `RDS` database. And you will need to install an npm package called `pg` locally on your development server. **NOTE:** The `pg` package install is only necessary if you are using **PostgresSQL** as your database.
+You will need to update the `database.json` file to configure Strapi to connect to the `RDS` database. And you will need to install a npm package called `pg` locally on your development server. **NOTE:** The `pg` package install is only necessary if you are using **PostgresSQL** as your database.
```

Comment:
I understand why you switched to an "a", but "an npm package" is correct. Please see https://www.npmjs.com/package/npm-install-package - the offical maintainers refer to it this way, and here is the grammar rule: https://www.quickanddirtytips.com/education/grammar/a-versus-an <--- This article talks about the _**_sound_**_ that the word has...